.coveragerc

```diff
@@ -487,7 +487,6 @@ omit =
     homeassistant/components/goodwe/sensor.py
     homeassistant/components/google_cloud/tts.py
     homeassistant/components/google_maps/device_tracker.py
-    homeassistant/components/google_pubsub/__init__.py
```

Comment:
Interestingly, this caused coverage failures on partial runs.
There were tests, but all integration files (there is only one) were ignored, and thus failed to generate a coverage report:
`WARNING: Failed to generate report: No data to report.`
See https://github.com/home-assistant/core/actions/runs/9791260278/job/27034757265